docs/experiments-msmarco-passage-subset.md

```diff
@@ -46,6 +52,24 @@ Next, we extract the contents into `data`.
 unzip data/msmarco_ans_small.zip -d data
 ```
 
+We should have these files in `data/msmarco_ans_small/`
+```
+ls data/msmarco_ans_small -1
+qrels.dev.small.tsv
+queries.dev.small.tsv
+run.dev.small.tsv
+scores
+``` 
+
+Let's also download MS MARCO passage dataset to visualize the actual passages after re-ranking.
```

Comment:
Are we only downloading it to visualize it?

Reply:
Yes, since the `data/msmarco_ans_small` itself does not include the passages. Any suggestion on how we can only download the passages corresponding to `data/msmarco_ans_small`?

Reply:
How were we reranking without this though?